locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,308 @@
+# English texts for the MOOC-Grader
+# Copyright (C) 2021 Ella Anttila
+# This file is distributed under the same license as the PACKAGE package.
```

Comment:
The PACKAGE placeholder could be replaced with MOOC-Grader. I can change this myself.